package.json

```diff
@@ -156,11 +156,16 @@
     "@breejs/later": "4.2.0",
     "@cdktf/hcl2json": "0.20.11",
     "@opentelemetry/api": "1.9.0",
+    "@opentelemetry/auto-instrumentations-node": "0.57.0",
```

Comment:
```suggestion
```
We have agreed not to add the Node auto instrumentation package

Reply:
☝️